caps/capabilities.go

```diff
@@ -107,3 +118,34 @@ func (r *Repository) Names() []string {
 	sort.Strings(keys)
 	return keys
 }
+
+// String representation of a capability
+func (c Capability) String() string {
+	return c.Name
+}
+
+type byName []Capability
+
+func (c byName) Len() int {
```

Comment:
Traditionally, all of that boilerplate is put in three sequential lines without any spacing:
```
func (c byName) Len() int { return len(c) }
func (c byName) Swap(...
```